lazy-pages/src/host_func.rs

```diff
@@ -88,10 +88,10 @@ impl<'a> AccessHandler for HostFuncAccessHandler<'a> {
         pages: Self::Pages,
         mut process_one: impl FnMut(GearPageNumber) -> Result<(), Error>,
     ) -> Result<(), Error> {
-        for page in pages {
-            process_one(page)?;
-        }
-        Ok(())
+        pages.iter().try_for_each(|page| -> Result<(), Error> {
```

Comment:
whether this one works really faster? Because it looks much less understandable. I cannot see any reasons to do like this

Reply:
https://doc.rust-lang.org/book/ch13-04-performance.html